js/admin.js

```diff
@@ -1381,6 +1381,20 @@ function ajaxStates(id_state_selected)
     }
   });
 
+  $.ajax({
+    url: 'index.php',
+    dataType: 'json',
+    cache: false,
+    data: 'token=' + state_token + '&ajax=1&contains_dni=1&tab=AdminAddresses&id_country=' + $('#id_country').val(),
+    success: function(resp) {
+      if (resp && resp.contains_dni) {
+        $("#contains_dni").fadeIn();
```

Comment:
@PierreRambaud Why do you use fadeIn/fadeOut while show/hide are used in bottom ?

Reply:
Same effect as States:
https://github.com/PrestaShop/PrestaShop/blob/1.7.6.x/js/admin.js#L1372

Reply:
:ok_hand: for me